operations/bills.md

```diff
@@ -204,6 +257,40 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 | `Iban` | string  | optional | Enterprise IBAN (International Bank Account Number). |
 | `Bic` | string  | optional | Enterprise BIC (Bank Identifier Code). |
 
+### Bill address
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Line1` | string  | optional | First line of the address. |
+| `Line2` | string  | optional | Second line of the address. |
+| `City` | string  | optional | City of the address. |
+| `Postal Code` | string  | optional | Postal code of the address. |
+| `SubdivisionCode` | string  | optional | ISO 3166-2 code of the administrative division. |
+| `CountryCode` | string  | optional | ISO 3166-1 code of the country. |
+
+#### Legal Identifiers
```

Comment:
@MikeAdamsMews FYI: Another example of the dictionary, key value pair.

Reply:
Ok, here's my proposal:

### Dictionary

Dictionary is a collection of key-value pairs.

| Property | Type | Contract | Description |
| :-- | :-- | :-- | :-- |
| ?Key? | string | optional | Some value corresponding to the ?Key? unique identifier. Cannot be null. |

### Legal Identifiers

`LegalIdentifiers` is a collection of key-value pairs, where the key is the type of legal identifier and the value is the corresponding value of that identifier. Keys are as follows:

* `TaxIdentifier`
* `CompanyName`
* `Siret`
* `Siren`
* `NafCode`
* `RcsCode`
* `LegalStatus`
* `RegisteredCapital`
* `CityOfRegistration`
* `TradesDirectoryRegistrationNumber`
* `ItDestinationCode`
* `ItFiscalCode`
* `ItLotteryCode`
* `HungarianVatCode`
* `HungarianCompanyName`
* `HungarianTaxPayerIdentifier`
* ...

Reply:
I will update language texts to also use Dictionary, and Dictionary will become a 'common object definition' and moved out of here to the new page of common object definitions, but this will be on a separate Pull Request. For the moment, it can be defined here.

Reply:
- I would remove Company name as we have already PR prepared that will remove it from Legal identifiers.
- Also values of this dictionary can be empty, for example `TradesDirectoryRegistrationNumber` is optional.

Reply:
@sianovd good idea. Removed the Company Name.
@MikeAdamsMews I've amended the text a bit to also reference to the new definition of Dictionary